rollup-data-structure.md

```diff
@@ -1,10 +1,13 @@
 ## Rollup Data Structure
-The management contract implements a blockchain-like data structure to hold the rollups. Each rollup points to a parent rollup, and at any time there can be multiple competing sibling rollups. Similar to L1 blockchains, it is the responsibility of the individual L2 nodes to decide which sibling is valid. The difference is that un-hacked nodes are not able to build upon an invalid rollup.
+The management contract implements a blockchain-like data structure to store the rollups. Each rollup points to a parent rollup, and at any time there can be multiple competing sibling rollups. Similar to L1 blockchains, it is the responsibility of the individual L2 nodes to decide which sibling is valid. The difference is that un-hacked nodes are running the same code and thus are not able to build upon an invalid rollup, or even to build on different rollups.
 
-The diagram below depicts a L1 blockchain (in black). Each block contains a snapshot of the state of the Obscuro rollup chain (in Red). There are 5 Obscuro nodes, each of them connected to different L1 nodes.
+The diagram below depicts a L1 blockchain (in black). Each block contains a snapshot of the state of the Obscuro rollup chain (in Red). In this example, there are 5 Obscuro nodes.
 
 ![block rollup simple](./images/block-rollup-simple.png)
 
-Note that forks are possible at both layers, and it is the responsibility of the Obscuro nodes to navigate and choose the most likely L1 block and L2 rollup. The following diagram shows two more complex scenarios, where firstly an extra aggregator publishes an additional rollup to a block, and secondly the L1 chain forks. 
+Note that forks are possible on both layers, and it is the responsibility of the Obscuro nodes to navigate and choose the most likely L1 block. The following diagram depicts a couple of complex scenarios, where firstly an extra aggregator publishes an additional rollup to a block, and secondly the L1 chain forks. 
 
-![block rollup complex](./images/block-rollup-complex.png)
\ No newline at end of file
+![block rollup complex](./images/block-rollup-complex.png)
+
+Note that this diagram will be dissected more after we introduce the consensus protocol. At this moment, it intends to give a high-level mental model of the decentralised nature of the L2 data model on top of the decentralised nature of the L1.
```

Comment:
I don't recall seeing this diagram dissected further and the consensus protocol is now POBI. Remove this paragraph?

Reply:
good point. I will add it to the pobi page and explain it there. It's a good diagram, but the reader doesn't have the tools to understand it in this chapter